fromgplus/fromgplus.php

```diff
@@ -38,8 +39,7 @@ function fromgplus_uninstall() {
 	Addon::unregisterHook('addon_settings_post', 'addon/fromgplus/fromgplus.php', 'fromgplus_addon_settings_post');
 }
 
-function fromgplus_addon_settings(&$a,&$s) {
-
+function fromgplus_addon_settings(App $a, $s) {
```

Comment:
The `&` operator of the `$s` parameter enables this function to change the hook data. If you remove it this function has no effect whatsoever.